config/sidekiq.yml

```diff
@@ -2,3 +2,4 @@
 :queues:
   - default
   - low
+  - retro
```

Comment:
Is this correct?

Reply:
Yes. We need to add it back in so that if anyone does a one-off retro job it will be processed.